packages/mutter.rb

```diff
@@ -0,0 +1,120 @@
+# Adapted from Arch Linux mutter PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/mutter/trunk/PKGBUILD
+
+require 'package'
+
+class Mutter < Package
+  description 'A window manager for GNOME'
+  homepage 'https://gitlab.gnome.org/GNOME/mutter'
+  version '40.beta'
+  compatibility 'x86_64 , armv7l'
+  source_url 'https://download.gnome.org/core/40/40.beta/sources/mutter-40.beta.tar.xz'
```

Comment:
`source_url "https://download.gnome.org/core/40/#{version}/sources/mutter-#{version}.tar.xz"`?

Reply:
It's a beta URL, they might change the final release URL config, so it doesn't make sense to overautomate this...